src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ReportsController.java

```diff
@@ -73,4 +78,22 @@ public ZipFilesSummaryReportListResponse getZipFilesSummary(
                 .collect(toList())
         );
     }
+
+    @GetMapping(path = "/zip-files-summary-csv", produces = MediaType.APPLICATION_OCTET_STREAM_VALUE)
+    @ApiOperation("Retrieves zip files summary report in csv format for the given date and jurisdiction")
+    public ResponseEntity downloadZipFilesSummary(
+        @RequestParam(name = "date") @DateTimeFormat(iso = DATE) LocalDate date,
+        @RequestParam(name = "jurisdiction", required = false) String jurisdiction
+    ) throws IOException {
+        List<ZipFileSummaryResponse> summary = this.reportsService.getZipFilesSummary(date, jurisdiction);
```

Comment:
is it possible to re-use same `ZipFilesSummaryReportListResponse` from method above?